projectq/meta/_control.py

```diff
@@ -101,15 +112,49 @@ def __init__(self, engine, qubits):
             qubits = [qubits]
         self._qubits = qubits
 
+        # If the user inputs a single digit 0 or 1, extend the digit to all ctrl qubits
```

Comment:
No! Now the cases for all zeroes or all ones are handled exclusively by the `State.AllOne`and `State.AllZero` enum values.
It just so happens that specifying `0` (interpreted as an integer) would have the same effect as `State.AllZero` but the value `1` should be the case of a single control qubit, similar to the case where `3` would be for 2 qubits.